src/style.css

```diff
@@ -0,0 +1,124 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+body {
+  font-weight: 300;
+  padding: 0;
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+.icon {
+  position: absolute;
+  bottom: 0;
+  left: 26px;
+}
+
+.iconSmall {
+  /* position: absolute; */
+  /* bottom: 0; */
```

Comment:
remove comments

Reply:
remove outline and ["jumping" on hover] - what do you mean?